### DISCUSSION TITLE
How to get FBA FC transfer information?

### DISCUSSION THREAD

**Discussion by Holdinar** [abuzuhri/Amazon-SP-API-CSharp#574] [Q&A]
I need help with Fulfillment Center (FC) transfers. Our objective is to identify the specific stock that is currently being transferred between FCs, and to trace the path from the source warehouse to the destination warehouse. Despite downloading numerous reports and making several requests, I am still unable to access the required information. If anyone could provide help or guidance, it would be greatly appreciated.

**Comment by ProNotion** [comment]
Looking at the docs there it seems like there may be data in the inventory ledger report for `InTransitBetweenWarehouses`. 

https://developer-docs.amazon.com/sp-api/docs/fba-inventory-reports-attributes#get_ledger_summary_view_data

I currently use the Inventory Ledger report for other purposes and don't remember seeing what you are asking for in this project but if it's supported by the API and just not implemented you could perhaps make a pull request?

**Reply by Holdinar** [reply]
Thanks for the anwser, I checked the report sadly only gives numbers.

**Reply by ProNotion** [reply]
`
GET_LEDGER_DETAIL_VIEW_DATA` Gives you the Fulfillment Center? Unfortunately, it may be a case of having to make many different calls and then piecing it all together to get the complete picture. It seems to be something frustratingly common with this API at the moment.

**Reply by Holdinar** [reply]
I think I might have found the proper report, but it does not seem to work nor there's any information on it.
https://developer-docs.amazon.com/sp-api/docs/report-type-values-tax

On Demand Stock Transfer Report
reportType value:
GET_GST_STR_ADHOC

Know anything about it?

**Comment by ProNotion** [comment]
For what marketplace? The description sounds like it would only apply to a marketplace such as the U.S. and not the U.K. which is where I am selling. I am no expert nor do I have experience in the U.S. marketplace but it is my understanding that the movement of stock within the UK does not create the same taxable events as I believe it does in the USA when crossing state lines

**Reply by Holdinar** [reply]
Europe mostly germany,spain,italy, france so on.

I also tried inventory ledger but it's very misleading, to figure out source and destination.
